diff --git a/dist/pool/index.mjs b/dist/pool/index.mjs
index ce77319945eadfdcd80326618a3f5153feee50d0..b4cc8d2630edac2130787dc7c5148a5cdd435011 100644
--- a/dist/pool/index.mjs
+++ b/dist/pool/index.mjs
@@ -63273,6 +63273,11 @@ async function buildProjectWorkerOptions(project, customOptions, main) {
 			type: "ESModule",
 			path: path.join(modulesRoot, "node:vm"),
 			contents: fs.readFileSync(path.join(DIST_PATH, `worker/node/vm.mjs`))
+		},
+		{
+			type: "ESModule",
+			path: path.join(modulesRoot, "cloudflare:test-internal"),
+			contents: fs.readFileSync(path.join(DIST_PATH, `worker/lib/cloudflare/test-internal.mjs`))
 		}
 	];
 	const workers = [runnerWorker];
